ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -211,12 +211,11 @@ export const useNewMhrRegistration = () => {
   }
 
   const fetchMhRegistrations = async (): Promise<void> => {
+    const isSummary = true // Used to indicate whether api should return registrations collapsed
```

Comment:
Not really required, as passing `true` argument in the method would display the argument name/descriptor anywho i think yea?

Reply:
I was conflicted... the only reason I added the const is for some explanation.... I will change...

Reply:
Absolutely, i appreciate that thought process!

Reply:
I think the best place for a comment on an argument for this would be in a doc-block comment on the root method itself. 
Thats where you could describe the `withCollapse` argument best i think.

Reply:
@ChasBoucher

Reply:
Thank you!